tests/system/fim/scenarios/203_whodata_frequency/config/agent_windows_ossec.conf

```diff
@@ -72,9 +72,9 @@
 
     <!-- Frequency that syscheck is executed default every 12 hours -->
     <frequency>43200</frequency>

```

Comment:
Please check the linux analog config

Reply:
Checked, both are correctly set to 43200.

Cheers !